ansible/devutil/inv_helpers.py

```diff
@@ -138,7 +138,7 @@ def get_host_creds(self, hostname):
         res["console_user"] = {}
         res["console_password"] = {}
 
-        for k, v in console_login_creds.iteritems():
+        for k, v in console_login_creds.items():
```

Comment:
Why do we need this change?

Reply:
yea, the `iteritems` function is deprecated in python3, and the recommendation is use `items` instead. (and it works XD)